mongodb/intro/query-operators/why-objectids-are-better-than-manually-inputted-ids.md

```diff
@@ -0,0 +1,70 @@
+﻿---
+author: Stefan-Stojanovic
+type: normal
+category: must-know
+---
+
+# Why Are `ObjectId`s Better Than Manually Inputted `_id`s?
+
+---
+
+## Content
+
+### 

```

Comment:
we should remove this empty headline. I think got copied with the rest of the insight
```suggestion
```